src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -288,6 +320,28 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		}
 	}
 
+	private func gatherTestResultMetadata() -> [SAP_Internal_Ppdd_PPATestResultMetadata] {
+		let metadata = store.testResultMetadata
+		var protoBuf = SAP_Internal_Ppdd_PPATestResultMetadata()
+		
+		if let testResult = metadata?.testResult?.protobuf {
+			protoBuf.testResult = testResult
+		}
+		if let hoursSinceTestRegistration = metadata?.daysSinceMostRecentDateAtRiskLevelAtTestRegistration {
+			protoBuf.hoursSinceTestRegistration = Int32(hoursSinceTestRegistration)
+		}
+		if let riskLevel = metadata?.riskLevelAtTestRegistration?.protobuf {
+			protoBuf.riskLevelAtTestRegistration = riskLevel
+		}
+		if let daysSinceMostRecentDateAtRiskLevelAtTestRegistration = metadata?.daysSinceMostRecentDateAtRiskLevelAtTestRegistration {
+			protoBuf.daysSinceMostRecentDateAtRiskLevelAtTestRegistration = Int32(daysSinceMostRecentDateAtRiskLevelAtTestRegistration)
+		}
+		if let hoursSinceHighRiskWarningAtTestRegistration = metadata?.hoursSinceHighRiskWarningAtTestRegistration {
+			protoBuf.hoursSinceHighRiskWarningAtTestRegistration = Int32(hoursSinceHighRiskWarningAtTestRegistration)
+		}
+
+		return [protoBuf]
```

Comment:
i am not really sure, but we used always the `.with` method to generate the protobuf.
So it could be written so:
```suggestion
		return SAP_Internal_Ppdd_PPATestResultMetadata().with {
		
		if let testResult = metadata?.testResult?.protobuf {
			$0.testResult = testResult
		}
		if let hoursSinceTestRegistration = metadata?.daysSinceMostRecentDateAtRiskLevelAtTestRegistration {
			$0.hoursSinceTestRegistration = Int32(hoursSinceTestRegistration)
		}
		if let riskLevel = metadata?.riskLevelAtTestRegistration?.protobuf {
			$0.riskLevelAtTestRegistration = riskLevel
		}
		if let daysSinceMostRecentDateAtRiskLevelAtTestRegistration = metadata?.daysSinceMostRecentDateAtRiskLevelAtTestRegistration {
			$0.daysSinceMostRecentDateAtRiskLevelAtTestRegistration = Int32(daysSinceMostRecentDateAtRiskLevelAtTestRegistration)
		}
		if let hoursSinceHighRiskWarningAtTestRegistration = metadata?.hoursSinceHighRiskWarningAtTestRegistration {
			$0.hoursSinceHighRiskWarningAtTestRegistration = Int32(hoursSinceHighRiskWarningAtTestRegistration)
		}
```